app/javascript/packages/document-capture/components/in-person-location-step.spec.tsx

```diff
@@ -7,8 +7,8 @@ import { setupServer } from 'msw/node';
 import { rest } from 'msw';
 import type { SetupServerApi } from 'msw/node';
 import AnalyticsContext, { AnalyticsContextProvider } from '../context/analytics';
-import InPersonLocationStep, { LOCATIONS_URL } from './in-person-location-step';
-import { ADDRESS_SEARCH_URL } from './address-search';
+import InPersonLocationStep from './in-person-location-step';
+import { ADDRESS_SEARCH_URL, LOCATIONS_URL } from './address-search';
```

Comment:
i wonder if we want to save this change for the refactor ticket as it seems related to that work?

Reply:
Yeah, that seems completely reasonable. Also, I'm not sure this is the best way to address this refactor, so I'm happy to undo it and save the change for a later ticket.